src/main/webapp/app/genomic/geneset.model.ts

```diff
@@ -0,0 +1,23 @@
+export interface Geneset {
+    name: string,
+    id: number,
+    genes: Gene[]
+}
+
+export interface Gene {
+    entrezGeneId: number,
+    hugoSymbol: string,
+    name: string,
+    oncogene: boolean,
+    curatedIsoform: string,
+    curatedRefSeq: string,
+    geneAliases: string[],
+    genesets: string[],
```

Comment:
I don't think this type is correct and you should not include this property in Gene to prevent the circular references.

Reply:
Removed genesets in Gene model